### ISSUE TITLE
Facility COVID status: CMS UX changes

### ISSUE THREAD

**Issue by cmaeng** [department-of-veterans-affairs/va.gov-cms#8960]

**Comment by rachel-kauff** [comment]
By May 13:
- [ ] Draft Editorial experience for VAMC editors (VAMC facility content type) [in Figma (WIP)](https://www.figma.com/file/W4oO1UO0ilSgHu1kXf1Sj0/VAMC-facilities---COVID-status?node-id=37%3A2448)
- [ ] Danielle to confirm guidelines and get approval from VHA digital media / COVID future state working group
- [ ] Finalize field group label, field group help text, and field help text, hand off to dev
- [ ] Draft KB articles and get review by Danielle (Operating status article(s) as well as separate article for COVID status)
- [ ] Get impromptu feedback from 2-3 PAOs (sprint 58) - outcomes roll into post-MVP iteration
- [ ] Publish KB content, align with training or other user-facing comms

**Comment by swirtSJW** [comment]
These came from @rachel-kauff   [in Slack](https://dsva.slack.com/archives/C0FQSS30V/p1651856057303499)

VAMC Facility node edit
![image](https://user-images.githubusercontent.com/5752113/167219343-dda2f5e9-70d0-4cd0-8f47-bbe370c9bbb0.png)

Selector
![image](https://user-images.githubusercontent.com/5752113/167219400-51807df6-8cfd-49a3-a7dd-b9762ded22f0.png)


Alternate option for if specific entry guideline is longer than ~100 chars
![image](https://user-images.githubusercontent.com/5752113/167219491-f0d78e11-9fb6-4fe2-9256-e29c96325a01.png)

**Comment by rachel-kauff** [comment]
UPDATE: We're going with radio buttons, so scratch the above. See [this thread for context](https://dsva.slack.com/archives/C0FQSS30V/p1652114200180059?thread_ts=1652109756.462559&cid=C0FQSS30V).

**Comment by suzanne-gray** [comment]
KB article [6869](https://prod.cms.va.gov/node/6869/edit) has been updated with COVID info and saved as draft. Needs approval.

**Comment by suzanne-gray** [comment]
KB article [6868](https://prod.cms.va.gov/node/6868/latest) has been updated with COVID info and saved as draft. Needs approval.

**Comment by EWashb** [comment]
@rachel-kauff is this ready to be closed or are we waiting on something else?

**Comment by rachel-kauff** [comment]
Ready to be closed! Anything new should prob get a new ticket.
